README.md

```diff
@@ -26,13 +26,13 @@ import module from "package/lib/module";
 
 There are problems with this approach:
 
-* it is often encouraging people to import files authored in CJS format, which
+- it is often encouraging people to import files authored in CJS format, which
   if produced with tools like [`babel`](https://github.com/babel/babel) has i.e.
   interop helper functions deoptimizing imported file size when comparing to the
   same file authored in ESM format. Also `webpack` just bails out on CJS files
   when trying to optimize your application size with techniques such as
   tree-shaking & scope hoisting (a.k.a module concatenation).
-* it is exposing **internal directory structure** to the user. Why `lib` is in
+- it is exposing **internal directory structure** to the user. Why `lib` is in
```

Comment:
Prettier changed this in the precommit hook. I'm not sure if they've changed their defaults or why it changed.

Reply:
i think the default might have changed, doesnt matter anyway - just keep this change as part of this PR